<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<<<<<<< HEAD
<testsuite ignored="0" hostname="AKK" failures="2" tests="14" name="Default test" time="80.469" errors="0" timestamp="2025-05-02T16:33:30 EDT">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.716"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.575"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.058"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.333"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.366"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.359"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="13.77">
    <failure type="org.openqa.selenium.NoSuchElementException" message="no such element: Unable to locate element: {&amp;quot;method&amp;quot;:&amp;quot;xpath&amp;quot;,&amp;quot;selector&amp;quot;:&amp;quot;//div[@class=&amp;apos;alert alert-primary&amp;apos;]&amp;quot;}
  (Session info: chrome=135.0.7049.115)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: &amp;apos;4.31.0&amp;apos;, revision: &amp;apos;1ef9f18787*&amp;apos;
System info: os.name: &amp;apos;Windows 11&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;23.0.2&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [16a7b04d4f3751c2e2415e20a5930722, findElement {value=//div[@class=&amp;apos;alert alert-primary&amp;apos;], using=xpath}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 135.0.7049.115, chrome: {chromedriverVersion: 135.0.7049.114 (63fd8a7d9d0..., userDataDir: C:\Users\skurr\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:49397}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:49397/devtoo..., se:cdpVersion: 135.0.7049.115, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 16a7b04d4f3751c2e2415e20a5930722">
      <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class='alert alert-primary']"}
  (Session info: chrome=135.0.7049.115)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.31.0', revision: '1ef9f18787*'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '23.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [16a7b04d4f3751c2e2415e20a5930722, findElement {value=//div[@class='alert alert-primary'], using=xpath}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 135.0.7049.115, chrome: {chromedriverVersion: 135.0.7049.114 (63fd8a7d9d0..., userDataDir: C:\Users\skurr\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:49397}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:49397/devtoo..., se:cdpVersion: 135.0.7049.115, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 16a7b04d4f3751c2e2415e20a5930722
at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:501)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:485)
at org.openqa.selenium.remote.ErrorCodec.decode(ErrorCodec.java:167)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:138)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:50)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:215)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:165)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:59)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:368)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:362)
at PageObject.RegistrationPageObject.errorMessageDisplayed(RegistrationPageObject.java:96)
at StepDefinitions.RegistrationPageStep.user_gets_error_message_for_username_in_the_register_page(RegistrationPageStep.java:92)
at âœ½.User gets error message for username in the Register page(file:///C:/Users/skurr/eclipse-workspace/ClubNinjas-dsAlgoBDD/src/test/resources/features/RegisterPage.feature:26)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.257"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.33"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.133"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.002"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="13.539">
    <failure type="org.openqa.selenium.NoSuchElementException" message="no such element: Unable to locate element: {&amp;quot;method&amp;quot;:&amp;quot;xpath&amp;quot;,&amp;quot;selector&amp;quot;:&amp;quot;//div[@class=&amp;apos;alert alert-primary&amp;apos;]&amp;quot;}
  (Session info: chrome=135.0.7049.115)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: &amp;apos;4.31.0&amp;apos;, revision: &amp;apos;1ef9f18787*&amp;apos;
System info: os.name: &amp;apos;Windows 11&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;23.0.2&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [831fe30737b79f5ce20422d7385328d7, findElement {value=//div[@class=&amp;apos;alert alert-primary&amp;apos;], using=xpath}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 135.0.7049.115, chrome: {chromedriverVersion: 135.0.7049.114 (63fd8a7d9d0..., userDataDir: C:\Users\skurr\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:49802}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:49802/devtoo..., se:cdpVersion: 135.0.7049.115, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 831fe30737b79f5ce20422d7385328d7">
      <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class='alert alert-primary']"}
  (Session info: chrome=135.0.7049.115)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.31.0', revision: '1ef9f18787*'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '23.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [831fe30737b79f5ce20422d7385328d7, findElement {value=//div[@class='alert alert-primary'], using=xpath}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 135.0.7049.115, chrome: {chromedriverVersion: 135.0.7049.114 (63fd8a7d9d0..., userDataDir: C:\Users\skurr\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:49802}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:49802/devtoo..., se:cdpVersion: 135.0.7049.115, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 831fe30737b79f5ce20422d7385328d7
at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:501)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:485)
at org.openqa.selenium.remote.ErrorCodec.decode(ErrorCodec.java:167)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:138)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:50)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:215)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:165)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:59)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:368)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:362)
at PageObject.RegistrationPageObject.errorMessageDisplayed(RegistrationPageObject.java:96)
at StepDefinitions.RegistrationPageStep.user_gets_an_error_message_for_password_in_the_register_page(RegistrationPageStep.java:126)
at âœ½.User gets an error message for password in the Register page(file:///C:/Users/skurr/eclipse-workspace/ClubNinjas-dsAlgoBDD/src/test/resources/features/RegisterPage.feature:47)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.622"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.462"/>
=======
<testsuite ignored="0" hostname="KAITLYN" failures="15" tests="85" name="Default test" time="642.538" errors="0" timestamp="2025-04-30T23:49:45 CDT">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="9.687"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.583"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.627"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.104"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.446">
    <failure type="java.lang.AssertionError" message="No Alert found">
      <![CDATA[java.lang.AssertionError: No Alert found
at org.testng.Assert.fail(Assert.java:111)
at PageObject.ArrayPageObject.acceptAlert(ArrayPageObject.java:152)
at StepDefinitions.ArrayPageStep.the_user_should_able_to_see_a_error_message_from_and(ArrayPageStep.java:126)
at âœ½.The user should able to see a error message from "TryCode" and 1(file:///C:/Users/jimre/eclipse-workspace/DSAlgoBDD_ClubNinjas/clubNinjas-BDD/src/test/resources/features/ArrayPage.feature:36)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.041"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.593"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.289"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.397"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.181"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.109"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.938"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.859"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.511"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.863"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="9.083">
    <failure type="java.lang.AssertionError" message="expected [Submission Successful] but found [ Some Tests failed. Please review code]">
      <![CDATA[java.lang.AssertionError: expected [Submission Successful] but found [ Some Tests failed. Please review code]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at StepDefinitions.ArrayPageStep.the_user_should_see_a_message(ArrayPageStep.java:234)
at âœ½.The user should see a message from "PracticeQns" and 6(file:///C:/Users/jimre/eclipse-workspace/DSAlgoBDD_ClubNinjas/clubNinjas-BDD/src/test/resources/features/ArrayPage.feature:99)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.106"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.901"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.451"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.098"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.69"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.528"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="9.034">
    <failure type="java.lang.AssertionError" message="expected [Submission Successful] but found [ Some Tests failed. Please review code]">
      <![CDATA[java.lang.AssertionError: expected [Submission Successful] but found [ Some Tests failed. Please review code]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at StepDefinitions.ArrayPageStep.the_user_should_see_a_message(ArrayPageStep.java:234)
at âœ½.The user should see a message from "PracticeQns" and 8(file:///C:/Users/jimre/eclipse-workspace/DSAlgoBDD_ClubNinjas/clubNinjas-BDD/src/test/resources/features/ArrayPage.feature:143)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.707"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.681"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.848"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.991"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.848"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.915"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.57"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.209"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.849"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.811"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.263"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.996">
    <failure type="java.lang.AssertionError" message="expected [Error occurred during submission] but found [ No tests were collected]">
      <![CDATA[java.lang.AssertionError: expected [Error occurred during submission] but found [ No tests were collected]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at StepDefinitions.ArrayPageStep.the_user_should_see_a_message(ArrayPageStep.java:234)
at âœ½.The user should see a message from "PracticeQns" and 3(file:///C:/Users/jimre/eclipse-workspace/DSAlgoBDD_ClubNinjas/clubNinjas-BDD/src/test/resources/features/ArrayPage.feature:221)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="9.348">
    <failure type="java.lang.AssertionError" message="expected [Error occurred during submission] but found [ No tests were collected]">
      <![CDATA[java.lang.AssertionError: expected [Error occurred during submission] but found [ No tests were collected]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at StepDefinitions.ArrayPageStep.the_user_should_see_a_message(ArrayPageStep.java:234)
at âœ½.The user should see a message from "PracticeQns" and 4(file:///C:/Users/jimre/eclipse-workspace/DSAlgoBDD_ClubNinjas/clubNinjas-BDD/src/test/resources/features/ArrayPage.feature:221)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="9.168">
    <failure type="java.lang.AssertionError" message="expected [Submission Successful] but found [ No tests were collected]">
      <![CDATA[java.lang.AssertionError: expected [Submission Successful] but found [ No tests were collected]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at StepDefinitions.ArrayPageStep.the_user_should_see_a_message(ArrayPageStep.java:234)
at âœ½.The user should see a message from "PracticeQns" and 12(file:///C:/Users/jimre/eclipse-workspace/DSAlgoBDD_ClubNinjas/clubNinjas-BDD/src/test/resources/features/ArrayPage.feature:232)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.142"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.947"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.508">
    <failure type="java.lang.AssertionError" message="No Alert found">
      <![CDATA[java.lang.AssertionError: No Alert found
at org.testng.Assert.fail(Assert.java:111)
at PageObject.ArrayPageObject.acceptAlert(ArrayPageObject.java:152)
at StepDefinitions.ArrayPageStep.the_user_should_able_to_see_a_error_message_from_and(ArrayPageStep.java:126)
at âœ½.The user should able to see a error message from "TryCode" and 1(file:///C:/Users/jimre/eclipse-workspace/DSAlgoBDD_ClubNinjas/clubNinjas-BDD/src/test/resources/features/ArrayPage.feature:250)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.123">
    <failure type="java.lang.AssertionError" message="No Alert found">
      <![CDATA[java.lang.AssertionError: No Alert found
at org.testng.Assert.fail(Assert.java:111)
at PageObject.ArrayPageObject.acceptAlert(ArrayPageObject.java:152)
at StepDefinitions.ArrayPageStep.the_user_should_able_to_see_a_error_message_from_and(ArrayPageStep.java:126)
at âœ½.The user should able to see a error message from "TryCode" and 2(file:///C:/Users/jimre/eclipse-workspace/DSAlgoBDD_ClubNinjas/clubNinjas-BDD/src/test/resources/features/ArrayPage.feature:250)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.074">
    <failure type="java.lang.AssertionError" message="No Alert found">
      <![CDATA[java.lang.AssertionError: No Alert found
at org.testng.Assert.fail(Assert.java:111)
at PageObject.ArrayPageObject.acceptAlert(ArrayPageObject.java:152)
at StepDefinitions.ArrayPageStep.the_user_should_able_to_see_a_error_message_from_and(ArrayPageStep.java:126)
at âœ½.The user should able to see a error message from "TryCode" and 3(file:///C:/Users/jimre/eclipse-workspace/DSAlgoBDD_ClubNinjas/clubNinjas-BDD/src/test/resources/features/ArrayPage.feature:250)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.663"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.315"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.987"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.484"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.363">
    <failure type="java.lang.AssertionError" message="No Alert found">
      <![CDATA[java.lang.AssertionError: No Alert found
at org.testng.Assert.fail(Assert.java:111)
at PageObject.ArrayPageObject.acceptAlert(ArrayPageObject.java:152)
at StepDefinitions.ArrayPageStep.the_user_should_able_to_see_a_error_message_from_and(ArrayPageStep.java:126)
at âœ½.The user should able to see a error message from "TryCode" and 1(file:///C:/Users/jimre/eclipse-workspace/DSAlgoBDD_ClubNinjas/clubNinjas-BDD/src/test/resources/features/ArrayPage.feature:285)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.656"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.354"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.215"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.997"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.75"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.639"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.523">
    <failure type="java.lang.AssertionError" message="No Alert found">
      <![CDATA[java.lang.AssertionError: No Alert found
at org.testng.Assert.fail(Assert.java:111)
at PageObject.ArrayPageObject.acceptAlert(ArrayPageObject.java:152)
at StepDefinitions.ArrayPageStep.the_user_should_able_to_see_a_error_message_from_and(ArrayPageStep.java:126)
at âœ½.The user should able to see a error message from "TryCode" and 1(file:///C:/Users/jimre/eclipse-workspace/DSAlgoBDD_ClubNinjas/clubNinjas-BDD/src/test/resources/features/ArrayPage.feature:320)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.176"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.701"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.1"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.571"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.576"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.396"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.117"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.275"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.782">
    <failure type="java.lang.AssertionError" message="No Alert found">
      <![CDATA[java.lang.AssertionError: No Alert found
at org.testng.Assert.fail(Assert.java:111)
at PageObject.ArrayPageObject.acceptAlert(ArrayPageObject.java:152)
at StepDefinitions.ArrayPageStep.the_user_should_able_to_see_a_error_message_from_and(ArrayPageStep.java:126)
at âœ½.The user should able to see a error message from "TryCode" and 1(file:///C:/Users/jimre/eclipse-workspace/DSAlgoBDD_ClubNinjas/clubNinjas-BDD/src/test/resources/features/QueuePage.feature:34)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.246"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.193"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.524"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.075"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.423"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.717"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.298">
    <failure type="java.lang.AssertionError" message="No Alert found">
      <![CDATA[java.lang.AssertionError: No Alert found
at org.testng.Assert.fail(Assert.java:111)
at PageObject.ArrayPageObject.acceptAlert(ArrayPageObject.java:152)
at StepDefinitions.ArrayPageStep.the_user_should_able_to_see_a_error_message_from_and(ArrayPageStep.java:126)
at âœ½.The user should able to see a error message from "TryCode" and 1(file:///C:/Users/jimre/eclipse-workspace/DSAlgoBDD_ClubNinjas/clubNinjas-BDD/src/test/resources/features/QueuePage.feature:70)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.147"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.463"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.253"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.125"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.325"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.426">
    <failure type="java.lang.AssertionError" message="No Alert found">
      <![CDATA[java.lang.AssertionError: No Alert found
at org.testng.Assert.fail(Assert.java:111)
at PageObject.ArrayPageObject.acceptAlert(ArrayPageObject.java:152)
at StepDefinitions.ArrayPageStep.the_user_should_able_to_see_a_error_message_from_and(ArrayPageStep.java:126)
at âœ½.The user should able to see a error message from "TryCode" and 1(file:///C:/Users/jimre/eclipse-workspace/DSAlgoBDD_ClubNinjas/clubNinjas-BDD/src/test/resources/features/QueuePage.feature:100)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.155"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.767"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.372"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.684"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.582"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.388">
    <failure type="java.lang.AssertionError" message="No Alert found">
      <![CDATA[java.lang.AssertionError: No Alert found
at org.testng.Assert.fail(Assert.java:111)
at PageObject.ArrayPageObject.acceptAlert(ArrayPageObject.java:152)
at StepDefinitions.ArrayPageStep.the_user_should_able_to_see_a_error_message_from_and(ArrayPageStep.java:126)
at âœ½.The user should able to see a error message from "TryCode" and 1(file:///C:/Users/jimre/eclipse-workspace/DSAlgoBDD_ClubNinjas/clubNinjas-BDD/src/test/resources/features/QueuePage.feature:130)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.548"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.972"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.673"/>
>>>>>>> master
</testsuite> <!-- Default test -->
